scripts/install

```diff
@@ -162,10 +162,10 @@ chown $app: $final_path/tmp
 #=================================================
 # SETUP SSOWAT
 #=================================================
-ynh_script_progression --message="Configuring SSOwat..." --weight=3
+ynh_script_progression --message="Configuring permissions..." --weight=3
 
 # Restrict access to admin only
-yunohost app addaccess --users=$admin $app
+ynh_permission_create --permission="admin" --allowed=$admin
 
```

Comment:
Eeeh I'm not sure the intention was to add an extra permission here ? I mean this permission has no corresponding URL or LDAP rule ?

Shouldn't it be more about removing `all_users` from the `main` permission, and just adding `$admin` ?

Reply:
Maybe we also want to define the main permission as protected to avoid mistakenly exposing it to visitors ?

Reply:
```
498  ERROR Permission phpmyadmin.main is protected. You cannot add or remove the visitors group to/from this permission.
```
Tests are failing because of the permissions, is there a way to make CI happy again?
https://ci-apps-dev.yunohost.org/ci/job/2382